NeoML/docs/en/Python/tutorials/IRNN.ipynb

```diff
@@ -260,7 +258,7 @@
    "id": "wooden-bubble",
    "metadata": {},
    "source": [
-    "But recurrent layers in NeoML usually return whole sequences. For experiment reproduction we must take only last elements of them. For this purpose we're gonna use `neoml.Dnn.SubSequence` layer."
+    "But recurrent layers in NeoML usually return whole sequences. To reproduce the experiment, we only need the last element of each. The `neoml.Dnn.SubSequence` layer will help us there."
```

Comment:
[asking-to-double-check] `will help us **there**`? Mb `here`? Or just write this sentence in another way...

Reply:
Certainly *here*, fixed.